credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt

```diff
@@ -331,9 +350,23 @@ class Web5TestVectorsCredentials {
       }
     }
 
-    testVectors.vectors.filter { it.errors ?: false }.forEach { vector ->
-      assertFails {
-        VerifiableCredential.verify(vector.input.vcJwt)
+    testVectors.vectors.filter { it.errors == true }.forEach { vector ->
+      if(vector.errorMessage == null || vector.errorMessage!!["web5-kt"] == null) {
```

Comment:
i'd rather not have error message be the deciding factor in whether our test is executed. is there another way?

Reply:
will update